composition-js/src/merging/merge.ts

```diff
@@ -2286,4 +2317,11 @@ class Merger {
         : err;
     });
   }
+
+  /**
+   * Get rid of leading '@' if present and return the list of directives passed to the Merger object
```

Comment:
In the same spirit of another comment, I'd rather we validate the user option and completely reject the case where a name doesn't start with a `@` rather than be flexible about it. I'd really worry we may regret that flexibility later.

Similarly, I'd have personally opted for, in that validation, maybe erroring out when one of the provided argument is simply not present in any subgraph.